spec/error_reaper_spec.rb

```diff
@@ -0,0 +1,49 @@
+module ExceptionHunter
```

Comment:
i feel like this was just thrown inside spec? maybe it could be located somewhere else

Reply:
Yeah but I'd like to have a good name for it. Right now there's no name that encapsulates all these modules